.github/workflows/unittests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v1
         with:
-          python-version: 3.8
+          python-version: 3.9
```

Comment:
They actually released 3.9 for github a few hours after your commit. Could you use the v2 version of the package though?

Reply:
Done!